Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/dao/UserLoginDAO.java

```diff
@@ -26,12 +26,13 @@
  */
 package gov.hhs.fha.nhinc.admingui.hibernate.dao;
 
+import java.util.List;
+
```

Comment:
My eclipse formatter is forcing that empty line to be inserted between the two group of imports.